projects/vdk-control-cli/src/vdk/internal/control/configuration/vdk_config.py

```diff
@@ -18,6 +19,11 @@ class VDKConfig:
 
     _op_id = os.environ.get("VDK_OP_ID_OVERRIDE", f"{uuid.uuid4().hex}"[:16])
 
+    _user_agent = os.environ.get(
+        "VDK_CONTROL_SERVICE_USER_AGENT",
+        f"vdk-control-cli/1.2 ({os.name}; {sys.platform})" + sys.version.split(" ")[0],
```

Comment:
You can get version from projects/vdk-control-cli/src/vdk/internal/control/command_groups/version_group/version.py


But how about this 
```suggestion
        f"vdk-control-cli/{version.__version__} ({sys.platform}; {platform.platform()}; ) Python {platform.python_version()} Node {platform.node()} User {getpass.getuser()}",
```

Reply:
I agree with adding `{version.__version__}`.


I also don't think we want to include the node name.
my node id is paulmxxxxxx.vmware.com'.
Seems like too much info to include in my opinion. 

@tozka Please let me know what you think.